src/pages/routes.test.ts

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs'
+import { parseStringPromise } from 'xml2js'
```

Comment:
nit: considering this is a test, you could just use the sync versions of readFile and parseString, and avoid any async